openjdk/excludes/ProblemList_openjdk24-openj9.txt

```diff
@@ -141,54 +157,29 @@ java/lang/Thread/virtual/MonitorWaitNotify.java#Xcomp-TieredStopAtLevel1-LM_LEGA
+java/lang/Thread/virtual/stress/TimedWaitALot.java#timeout-notify https://github.com/eclipse-openj9/openj9/issues/21426 generic-all
+java/lang/Thread/virtual/stress/TimedWaitALot.java#timeout-notify-interrupt https://github.com/eclipse-openj9/openj9/issues/21426 generic-all
 java/lang/ThreadGroup/SetMaxPriority.java https://github.com/eclipse-openj9/openj9/issues/6691 generic-all
 java/lang/Throwable/SuppressedExceptions.java https://github.com/eclipse-openj9/openj9/issues/6692 generic-all
 jni/nullCaller/NullCallerTest.java https://github.com/eclipse-openj9/openj9/issues/15370 linux-ppc64le,macosx-all
-java/lang/Thread/jni/AttachCurrentThread/AttachTest.java#id1 https://github.com/eclipse-openj9/openj9/issues/17399 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#LM_LEGACY https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#LM_LIGHTWEIGHT https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xcomp-LM_LEGACY https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xcomp-LM_LIGHTWEIGHT https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xcomp-TieredStopAtLevel1-LM_LEGACY https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xcomp-TieredStopAtLevel1-LM_LIGHTWEIGHT https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xcomp-noTieredCompilation-LM_LEGACY https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xcomp-noTieredCompilation-LM_LIGHTWEIGHT https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xint-LM_LEGACY https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#Xint-LM_LIGHTWEIGHT https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/MonitorEnterExit.java#default https://github.com/eclipse-openj9/openj9/issues/21422 generic-all
-java/lang/Thread/virtual/Starvation.java https://github.com/eclipse-openj9/openj9/issues/21423 generic-all
-java/lang/Thread/virtual/SynchronizedNative.java#Xcomp-TieredStopAtLevel1 https://github.com/eclipse-openj9/openj9/issues/21727 macosx-all,linux-ppc64le,aix-all
-java/lang/Thread/virtual/SynchronizedNative.java#Xcomp-noTieredCompilation https://github.com/eclipse-openj9/openj9/issues/21727 macosx-all,linux-ppc64le,aix-all
-java/lang/Thread/virtual/SynchronizedNative.java#Xint https://github.com/eclipse-openj9/openj9/issues/21727 macosx-all,linux-ppc64le,aix-all
-java/lang/Thread/virtual/SynchronizedNative.java#default https://github.com/eclipse-openj9/openj9/issues/21727 macosx-all,linux-ppc64le,aix-all
```

Comment:
These should not have been removed; the issue is still open and the test still fails; see https://openj9-jenkins.osuosl.org/job/Test_openjdk24_j9_sanity.openjdk_aarch64_mac_Personal/49.

Reply:
Created https://github.com/adoptium/aqa-tests/pull/6236